Main.java

```diff
@@ -1,5 +1,8 @@
 public class Main {
     public static void main(String[] args) {
```

Comment:
Consider renaming variable 'userAgent' to something more descriptive, like 'browserString'.